kola/tests/coretest/core.go

```diff
@@ -70,7 +70,7 @@ func init() {
 		NativeFuncs: map[string]func() error{
 			"PortSSH":          TestPortSsh,
 			"DbusPerms":        TestDbusPerms,
-			"ServicesActive":   TestServicesActiveRHCOS,
+			"ServicesActive":   TestServicesActiveFCOS,
```

Comment:
Let's rename the actual test function from `TestServicesActiveFCOS` to `TestServicesActiveCoreOS` since it's targeting both OS's now.

Reply:
done